content/english/python-interactive/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Python: Interacting with the Console"
+description: "Exploring Python's capabilities with the Console"
+date: 2019-07-25T13:24:17-07:00
+prereq: "Python Basics"
+difficulty: "Intermediate"
+image: ""
+draft: false
+icon: "fab fa-python"
+weight: 13
+translationKey: "python-interactive-home"
+---
+
+# Introduction
+
+To start this program, open up a new Repl.it document and create a main() function definition and function call.
+
```

Comment:
I also think that the workshop might be quite long, because there are many functions to implement. May I suggest having a starting repl.it with all the function stubs already implemented?

I also suggest perhaps having some kind of way to verify that the student has correctly implemented the functions. We don't have an example in master yet, but one of the UCSD interns created repl.its that were built specifically to validate correctness of functions - perhaps you can take inspiration from this?